src/GitHub.Exports/Services/IGitService.cs

```diff
@@ -7,6 +7,26 @@ namespace GitHub.Services
 {
     public interface IGitService
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LocalRepositoryModel"/> class.
+        /// </summary>
+        /// <param name="localPath">The repository's local path.</param>
+        /// <returns>A repository model.</returns>
+        ILocalRepositoryModel CreateLocalRepositoryModel(string localPath);
+
+        /// <summary>
+        /// Creates a new branch model for the current branch.
+        /// </summary>
+        /// <param name="model">The <see cref="ILocalRepositoryModel" /> to create a current branch model for.</param>
+        /// <returns>A new branch model.</returns>
```

Comment:
We could change it to `GetBranch(string name = "HEAD")`. Do you think that would make sense? Otherwise we'd probably need to qualify it with `GetHeadBranch` or something?

Reply:
Yeah I think that would make sense.